python/activator/middleware_interface.py

```diff
@@ -313,6 +307,7 @@ def _collection_template(self):
     # Class invariants:
     # self.image_host is a valid URI with non-empty path and no query or fragment.
     # self._download_store is None if and only if self.image_host is a local URI.
+    # self._skip_spatial_preload indicates that the coordinates cannot be trusted.
```

Comment:
This is not actually an invariant...

Reply:
Thanks for catching this. Wrong place to document...  😓

Reply:
Moved away and rephrased the comments.